vault/external_tests/quotas/quotas_test.go

```diff
@@ -231,7 +231,7 @@ func TestQuotas_RateLimitQuota_ExemptPaths(t *testing.T) {
 	ideal := 8 + (7.7 * float64(elapsed) / float64(time.Second))
 	want := int32(ideal + 1)
 	require.NotZerof(t, numFail, "expected some requests to fail; numSuccess: %d, elapsed: %d", numSuccess, elapsed)
-	require.Lessf(t, numSuccess, want, "too many successful requests;numSuccess: %d, numFail: %d, elapsed: %d", want, numSuccess, numFail, elapsed)
+	require.LessOrEqualf(t, numSuccess, want, "too many successful requests;numSuccess: %d, numFail: %d, elapsed: %d", want, numSuccess, numFail, elapsed)
```

Comment:
Looks like there are 3 %-placeholders and 4 args to be interpolated, can you fix that at the same time please?

Reply:
oh, sorry I forgot about that. Sure thing

Reply:
i wonder if that might have been why it was misprinted in CI - rather than give an error it just dumps what it has